src/Addresses.hpp

```diff
@@ -57,16 +57,15 @@ inline T (*MC2_PROC_PTR_VA(const std::uintptr_t address))(Types..., ...) {
 // Functions
 
 #define sub_5772E0 (MC2_PROC_PTR<void *, size_t, const char *, std::uint32_t>(0x005772E0))
-#define sub_612E10 (MC2_PROC_PTR<bool, char *>(0x00612E10))
-#define sub_612E30 (MC2_PROC_PTR<bool, char *, std::uint32_t, void *>(0x00612E30))
-#define sub_612F00 (MC2_PROC_PTR<void>(0x00612F00))
 #define sub_614A10 (MC2_PROC_PTR<char *, const char *, const char *, std::uint32_t>(0x00614A10))
-
 #define MC2_MALLOC(size) (sub_5772E0(size, "OpenMC2:" __FILE__, __LINE__))
+#define MC2_FREE(ptr) (MC2_PROC_PTR<void *>(0x00577320))
```

Comment:
This doesn't look right at all. The way I'd want it done anyways would be to create the `sub_577320` definition, and define `MC2_FREE` as `(sub_577320(ptr))`. That way, it would match the other macros around it.